iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -31,54 +31,118 @@ class RoudiFindService_test : public RouDiServiceDiscoveryTest
 
 TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
 {
+    auto isServiceOffered = senderRuntime->offerService({"service1", "instance1"});
+    this->InterOpWait();
+
+    auto instanceContainer = receiverRuntime->findService({"service1", "instance1"});
+
+    ASSERT_THAT(instanceContainer.get_value().size(), Eq(1u));
```

Comment:
The API of the expected changed with the last big PR from @elfenpiff. Now `get_value()` is marked as deprecated and will not be accepted by the strict build in iceoryx CI. 
Instead please use `value()` in every occurrence.